triton-client/src/components/ProjectCardList/index.tsx

```diff
@@ -1,35 +1,57 @@
 import "@components/Common.scss"
 import { useNavigate } from "react-router-dom"
-import { List, Typography, Avatar } from "antd"
+import { List, Typography, Avatar, Divider } from "antd"
 import { ProjectCard, ProjectCardListProps } from "./interfaces"
 import projectIcon from "@static/project-icon.png"
+import InfiniteScroll from "react-infinite-scroll-component"
 
 export default function ProjectCardList({ projects }: ProjectCardListProps) {
     const navigate = useNavigate()
     const { Title, Text } = Typography
     const items = projects
     return (
-        <List
-            className="mgc-projects-list"
-            size="large"
-            header={
-                <Title className="mgc-triton-user-projects" level={3}>
-                    Projects
-                </Title>
-            }
-            bordered
-            dataSource={items}
-            renderItem={(item: ProjectCard) => (
-                <List.Item
-                    onClick={() => navigate(`/project/${item.external_id}/`)}
-                    style={{ cursor: "pointer" }}
-                >
-                    <List.Item.Meta
-                        avatar={<Avatar src={projectIcon} />}
-                        title={<Text>{item.external_name}</Text>}
-                    />
-                </List.Item>
-            )}
-        />
+        <div
+            id="scrollableDiv"
+            style={{
+                height: "90vh",
+                overflow: "auto",
+                border: "1px solid rgba(140, 140, 140, 0.35)",
+            }}
+        >
+            <InfiniteScroll
+                loader={<Divider plain />}
+                dataLength={items.length}
+                hasMore={items.length < 100}
+                endMessage={<Divider plain>It is all, nothing more 🤐</Divider>}
```

Comment:
Use something more professional for the endMessage.

Reply:
😢